src/CloudApi/CloudCredentials.php

```diff
@@ -23,29 +24,37 @@ public function __construct(CloudDataStore $datastoreCloud) {
 
   /**
    * @return string|null
+   * @throws \Acquia\Cli\Exception\AcquiaCliException
    */
   public function getCloudAccessToken(): ?string {
-    if (getenv('ACLI_ACCESS_TOKEN')) {
-      return getenv('ACLI_ACCESS_TOKEN');
+    if ($token = getenv('ACLI_ACCESS_TOKEN')) {
+      return $token;
     }
 
-    if (getenv('ACLI_ACCESS_TOKEN_FILE')) {
-      return trim(file_get_contents(getenv('ACLI_ACCESS_TOKEN_FILE')), "\"\n");
+    if ($file = getenv('ACLI_ACCESS_TOKEN_FILE')) {
+      if (!file_exists($file)) {
+        throw new AcquiaCliException('Access token file not found at {file}', ['file' => $file]);
```

Comment:
@anavarre let me know if you want me to add a link to a known issue or any other information here

Reply:
Linked in the Jira ticket.